pages/gacha.html

```diff
@@ -77,8 +77,18 @@ <h1>The Regis Altare Birthday Project 2023</h1>
       </nav>
 
       <section id="gacha-controls">
-        <button id="gacha-button">Pull Gacha</button>
-        <section id="card-list"></section>
+        <form>
+          <label>Show as:</label>
+          <input type="radio" id="gacha-pile" name="gacha-display" value="gacha-pile">
+          <label for="gacha-pile"> Card Pile</label>
+          <input type="radio" id="gacha-grid" name="gacha-display" value="gacha-grid">
+          <label for="gacha-grid"> Card Grid</label>
+        </form>
+        <button id="gacha-button" type="submit">Pull Gacha</button>
+        <div id="gacha-notes">
+          <p>Re-pulling while half viewed, or flipping a card in 'pile' view will not allow you to see previous cards, but will be saved to your Collection. When the card has stopped moving, animate it again by clicking / hovering!</p>
+        </div>
+        <section id="card-list">Ain't nobody here but us Altventurers. Pull the gacha to get started!</section>
```

Comment:
Wording update is totally fine - this was just what was there by default and hadn't been changed.

The 'pull' button and the 'pull this gacha to get started' note are separated because the gacha then fully replaces it. Ideas I was playing with was hiding the light blue box altogether (so the text must be placed elsehwere); and only showing it when gacha is pulled.

Reply:
ok, sounds like it's not a super quick change to move the pull button, so I've added it to our Notion board as a nice to have